config/initializers/oauth2.rb

```diff
@@ -0,0 +1,4 @@
+OAuth2::Response.register_parser(:text, 'text/plain') do |body|
+  key, value = body.split('=')
+  {key => value}
+end
```

Comment:
still need this or was upgrading the gem dependencies all that was needed here?

Reply:
Nah didn't need this anymore. Deleted